Integrations/GitHub/GitHub.py

```diff
@@ -0,0 +1,312 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+''' IMPORTS '''
+from datetime import datetime, timedelta
+import json
+import requests
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+''' GLOBALS/PARAMS '''
+
+
+API_KEY = demisto.params().get('apikey')
+OWNER = demisto.params().get('owner')
+REPO = demisto.params().get('repository')
+BASE_URL = 'https://api.github.com'
+FETCH_INTERVAL = demisto.params()['fetch_interval']
+
+
+''' HELPER FUNCTIONS '''
+
+
+def http_request(method, url_suffix, json=None, params=None, command=None):
+    if method == 'GET':
+        headers = {}
+    elif method in ('POST', 'PATCH'):
+        if not API_KEY:
+            headers = {
+                'Content-Type': 'application/json',
+                'Accept': 'application/vnd.github.v3+json'
+            }
+        else:
+            headers = {
+                'Content-Type': 'application/json',
+                'Accept': 'application/vnd.github.v3+json',
+                'Authorization': API_KEY
+            }
+
+    try:
+        res = requests.request(
+            method,
+            BASE_URL + url_suffix,
+            data=json,
+            headers=headers,
+            params=params
+        )
+
+        if res.status_code == 410:
+            return_error("Error - this issue was deleted")
+
+        elif res.status_code == 404:
+            return_error("Error - this issue doesn't exists ")
+
+        return res.json()
+
+    except Exception as e:
+        raise(e)
+
+
+def fetch_incidents():
+    now_utc = datetime.utcnow()
+    last_run_data = demisto.getLastRun()
+    last_run_time = last_run_data.get('time')
+
+    if last_run_time is None:
+        last_run_time = now_utc - timedelta(minutes=int(FETCH_INTERVAL))
+    else:
+        last_run_time = datetime.strptime(last_run_time, '%Y-%m-%dT%H:%M:%SZ')
+
+    all_issues = List_all_issues('false')
+
+    incidents = []
+
+    for issue in all_issues:
+        issue_create_time = issue['created_at']
+        issue_create_time = datetime.strptime(issue_create_time, "%Y-%m-%dT%H:%M:%SZ")
+
+        if issue_create_time > last_run_time:
+            incidents.append({
+                'name': str(issue['number']) + " - GitHub integration",
+                'occurred': issue['created_at'],
+                'rawJSON': json.dumps(issue)
+            })
+
+        if issue_create_time > last_run_time:
+            last_run_time = issue_create_time
+
+    demisto.incidents(incidents)
+    demisto.setLastRun({'time': last_run_time.isoformat().split('.')[0] + 'Z'})
+
+
+''' COMMANDS + REQUESTS FUNCTIONS '''
+
+
+def get_issue_context(issue):
+    label_context = []
+    for l in issue['labels']:
```

Comment:
l ?
Please provide a better name

Reply:
maybe try to do it in a one liner as well
label_context = [label['name'] for label in issue['labels']]